wandb/sdk/interface/interface.py

```diff
@@ -440,6 +440,24 @@ def publish_link_artifact(
     def _publish_link_artifact(self, link_artifact: pb.LinkArtifactRecord) -> None:
         raise NotImplementedError
 
+    def publish_use_artifact(
+        self,
+        artifact: Artifact,
+    ) -> None:
+        # use_artifact is either a public.Artifact or a wandb.Artifact that has been
+        # waited on and has an id
+        assert artifact.id is not None, "Artifact must have an id"
+        use_artifact = pb.UseArtifactRecord()
+        use_artifact.id = artifact.id
+        use_artifact.type = artifact.type
+        use_artifact.name = artifact.name
```

Comment:
total nit, but you could do this:
```suggestion
        use_artifact = pb.UseArtifactRecord(    
            id=artifact.id, type=artifact.type, name=artifact.name
        )
```

Reply:
Nice!